test_bundle_image.sh

```diff
@@ -174,11 +174,13 @@ function start_container {
 			CONTAINER_HOSTNAME=$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gateway}}')
 			CONTAINER_HTTP_PORT=8081
 
-			echo -e "web.server.host=${CONTAINER_HOSTNAME}\nweb.server.http.port=8081" > portal-ext.properties
+			echo -e "web.server.host=${CONTAINER_HOSTNAME}\nweb.server.http.port=8081" > "${TEST_DIR}/portal-ext.properties"
+
+			cp --recursive "${TEST_DIR}" "/opt/dev/projects/github/liferay-docker"
```

Comment:
@kiwm, will the `TEST_DIR` be deleted once the `test_bundle_image.sh` finishes to execute?

I mean, will the https://github.com/natocesarrego/liferay-docker/blob/master/release/post_build_clean.sh take care of it?